### DISCUSSION TITLE
Surely this obvious feature isn't missing, right?

### DISCUSSION THREAD

**Discussion by binvius** [garywill/BigSearch#15] [Ideas]
As a successful entrepreneur, I have the joy of being able to very healthily reward good devs who create useful tools. I was just about to buy you a considerable amount of coffee but was stopped in my tracks.

Am I right in thinking that if users want to add more to the side-categories such as 'Programming' and 'Multimedia' etc. they can only instead add their own additions to the side-category 'User Custom?'

Not being able to simply edit the side-categories in the same fashion that the 'User Custom' side-category is (via 'Edit custom engines' buttons for each side-category) seems like such an obvious oversight that I would be surprised if it is not me who is, in fact, missing something as the plugin is rendered unusable by such a large ratio of users as a result of missing such an obvious feature.

Please tell me I'm wrong or you are at least planning on rectifying this as soon as possible as there will be so few users in the world that have exactly the same search engine requirements as you. 

Happy to drop a load of cash if things are fixed as quickly as possible.

Thanks for the work you've done so far.

**Comment by garywill** [comment]
Currently user can only customize "User Custom" catagory.
In the future I plan to allow user to add their own sub-catagories, under "User Custom".

**Reply by binvius** [reply]
Thanks for the reply.

It sounds like this is just a project that you have built for yourself and not designed for other people, which is a shame.

Other users will obviously have different requirements than the limited ones you personally use so not being able to edit the side-categories, renders the project useless for about 99.9% of potential users. 

One really quick and easy way to resolve this fatal issue would be to simply copy the code used to provide the "User Custom" category into each of the other categories to allow each of them to be fully customisable as well. That should only take a few minutes to do which would then open the project up to the other 99.9% of users you can not currently use it due to its limitations.

Feel free to let me know if you ever decide to open this project up to users other than yourself as like I said, I am always happy to heavily financially reward good devs so it would very much be in your interest.
